src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -822,6 +822,12 @@ public List<FacetLabel> getFileTagsFacetLabels() {
      * @return boolean
      */
     public boolean isThisVersionSearchable() {
+        // Just like on the collection page, facets on the Dataset page can be
+        // disabled instance-wide by an admin:
+        if (settingsWrapper.isTrueForKey(SettingsServiceBean.Key.DisableSolrFacets, false)) {
+            return false;
+        }
+        
```

Comment:
re: the following code - is it worth it to query solr just to see if you can query solr? Would a check of the last indexed time be a reasonable alternative? (Not sure how much this would save given it's a simple query).

Reply:
Yeah, this looks like it's yet another thing we can save/economize on. 
We are already checking that this is the latest released version...  so, yes, comparing the version published date against the indexed time stamp should be reliable enough. 
These queries that just search for the version id cannot be that expensive; but there's definitely a lot of them in the logs.